hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -445,7 +444,18 @@ public void removeBucketAcls(
   @Override
   public Token<OzoneTokenIdentifier> getDelegationToken(Text renewer)
       throws IOException {
-    return ozoneManagerClient.getDelegationToken(renewer);
+
+    Token<OzoneTokenIdentifier> token =
+        ozoneManagerClient.getDelegationToken(renewer);
+    if (token != null) {
+      Text dtService =
+          getOMProxyProvider().getProxy().getDelegationTokenService();
+      token.setService(dtService);
+      LOG.info("Created " + token.toString());
+    } else {
+      LOG.info("Cannot get ozone delegation token from " + renewer);
```

Comment:
Shall we replace "Cannot get ozone delegation token from" with "Cannot get ozone delegation token for {}, renewer"?

Reply:
will fix in the next push.